pyproject.toml

```diff
@@ -23,7 +23,7 @@ classifiers = [
   'Topic :: Scientific/Engineering :: Astronomy',
   'Topic :: Scientific/Engineering :: Physics'
 ]
-dependencies = ["numpy>=1.21.2", "scipy"]
```

Comment:
simweights works perfectly with numpy2 and the latest version of nuflux contains the 'numpy<2' requirement so this is unneeded